src/pickler/parser.clj

```diff
@@ -88,6 +93,14 @@
   [_ bb]
   {:type :tuple})
 
+(defmethod opcode 0x86
+  [_ bb]
+  {:type :tuple2})
```

Comment:
Thinking about it again, there shouldn't be a difference anymore between TUPLE and TUPLE2 in the AST.